client/src/components/Admin/Products/UserList.js

```diff
@@ -74,91 +71,91 @@ const UserList = ({}) => {
                       </tr>
                     </thead>
                     <tbody>
-                     {display.map(display=>( 
-                      <tr class="candidates-list">
-                        <td class="title">
-                          <div class="thumb">
-                            <img
-                              class="img-fluid"
-                              src="https://bootdey.com/img/Content/avatar/avatar7.png"
-                              alt=""
-                            />
-                          </div>
-                          <div class="candidate-list-details">
-                            <div class="candidate-list-info">
-                              <div class="candidate-list-title">
-                                <h5 class="mb-0">
-                                  <a href="#">{display.first_name}</a>
-                                </h5>
-                              </div>
-                              <div class="candidate-list-option">
-                                <ul class="list-unstyled">
-                                  <li>
-                                    <i class="fas fa-filter pr-1"></i>
-                                    {display.username}
-                                  </li>
-                                  <li>
-                                    <i class="fas fa-map-marker-alt pr-1"></i>
-                                    {display.email}
-                                  </li>
-                                </ul>
+                      {display.map((display) => (
+                        <tr class="candidates-list">
+                          <td class="title">
+                            <div class="thumb">
+                              <img
+                                class="img-fluid"
+                                src="https://bootdey.com/img/Content/avatar/avatar7.png"
```

Comment:
can remove this image and simple use a placeholder avatar from fontawesome or bootstrap?